docs/guides/dropout.rst

```diff
@@ -18,84 +16,39 @@ Flax ``Dropout``.
   import jax
   import optax
 
-Split the PRNG key
+PRNGs in JAX
 ******************
 
-Since dropout is a random operation, it requires a pseudorandom number generator
-(PRNG) state. Flax uses JAX's (splittable) PRNG keys, which have a number of
-desirable properties for neutral networks. To learn more, refer to the
-`Pseudorandom numbers in JAX tutorial <https://jax.readthedocs.io/en/latest/jax-101/05-random-numbers.html>`__.
-
-**Note:** Recall that JAX has an explicit way of giving you PRNG keys:
-you can fork the main PRNG state (such as ``key = jax.random.PRNGKey(seed=0)``)
-into multiple new PRNG keys with ``key, subkey = jax.random.split(key)``. You
-can refresh your memory in
-`🔪 JAX - The Sharp Bits 🔪 Randomness and PRNG keys <https://jax.readthedocs.io/en/latest/notebooks/Common_Gotchas_in_JAX.html#jax-prng>`__.
+Since dropout uses random masks, it requires a pseudorandom number generator
```

Comment:
The new explanation raises more questions for less experienced users. Let's keep original text. 

Example: "dropout uses random masks".

Assume not every researcher is proficient in randomness in deep learning, basically.